### PR TITLE
Fix issue #265

### PR DESCRIPTION
**Pull request by GregSavin**
Wait for fileOpenHandler to finish, before refreshing the trace explorer context.

### CODE REVIEW

**Comment by marcdumais-work** [comment]
> Thanks for the contribution. In order to merge this PR you need to sign the ECA as described in the [contribution guide](https://github.com/eclipse-cdt-cloud/vscode-trace-extension/blob/master/CONTRIBUTING.md#eclipse-contributor-agreement). Please follow the instructions there. Thanks.

The ECA check now passes

**Comment by marcdumais-work** [comment]
@bhufmann 
There is another similar case of a call to fileOpenHandler() without an "await", a bit down in the code in extension.ts:
https://github.com/eclipse-cdt-cloud/vscode-trace-extension/pull/267/files#diff-4decf63ca1dab51f824c7ff2fd1ffb65bc509e91da293715b643854255f7d76aR170

That one seems used when using the "Open Trace" command. That one seems to work correctly, but I';m not sure why?

**Review by marcdumais-work — APPROVED** [review]
The code change is small and elegant! I tested with my own extension that contributes a custom command that executes command "traces.openTraceFile" to open a trace file I have locally. I also tested without an extension, as suggested by @bhufmann in the comments on the corresponding issue. In both cases I was able reproduce the issue using the latest master branch, and in both cases, this PR branch fixed the problem. 

Thanks @GregSavin for this contribution ! I believe we will need a second committer approval before we can merge.

**Review by bhufmann — APPROVED** [review]
Looks good to me, too. Thanks!

**Comment by marcdumais-work** [comment]
@GregSavin I'm not sure if you consume this extension from the VS Marketplace and/or openvsx, but FYI I think we will likely do a release soon that includes your fix. Probably after the following PR is merged:
#262

**Comment by GregSavin** [comment]
Thanks all!

**Pull request merged**
